src/query/xasl_cache.c

```diff
@@ -114,7 +114,11 @@ struct xcache
   INT32 cleanup_flag;
   BINARY_HEAP *cleanup_bh;
   XCACHE_CLEANUP_CANDIDATE *cleanup_array;
-
+  int soft_limit;
+  int hard_limit;
+  int max_plan_size;
+  volatile INT32 memory_usage_cache;
+  volatile INT32 memory_usage_clone;
```

Comment:
```suggestion
  INT32 memory_usage_clone;
```
전역 memory_usage_cache, memory_usage_clone의 값을 변경할 때는 ATOMIC_INC_32 등의 함수를 사용합니다.
따라서 동시성이 보장된다고 볼 수 있다고 생각합니다.
만약 volatile 또한 동시성 보장을 위해 사용한 것이라면 빼는 것이 어떨까요?
다른 이유가 있어서 volatile을 사용했다면 그 이유를 알려주시면 감사하겠습니다.

Reply:
제거 완료하였습니다.